FinniversKit/Sources/Components/Ribbon/Models/RibbonViewModel.swift

```diff
@@ -4,12 +4,29 @@
 
 import UIKit
 
-public struct RibbonViewModel: Hashable {
+public struct RibbonViewModel: Hashable, Codable {
```

Comment:
Currently one of main concern for FinniverseKit SDK is everything related to UI.
All Codable models should be located inside FinnClient SDK as part of API.

If you need that model for UI as well as API, best option is to create 2 models and convert they if needed

Reply:
noted